content/sensu-go/6.1/release-notes.md

```diff
@@ -68,6 +69,50 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.1.0 release notes
+
+**October 5, 2020** &mdash; The latest release of Sensu Go, version 6.1.0, is now available for download.
+
+**PLACEHOLDER FOR RELEASE SYNOPSIS**
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.1.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][172]) Added support for custom secrets engine paths in [Vault secrets][175].
+- ([Commercial feature][172]) In the web UI, added [new search functionality][178], with improved syntax and suggestions.
+- ([Commercial feature][172]) Added [`strict` attribute][179] to the PostgresConfig type to help debug incorrect configurations and database permissions.
+- ([Commercial feature][172]) Added [`batch_buffer`, `batch_size`, and `batch_workers` attributes][181] to the PostgresConfig type so operators can optimize PostgreSQL latency and throughput.
+- ([Commercial feature][172]) Added TLS configuration to the cluster resource so you can specify additional CA certificates and insecure mode.
+- ([Commercial feature][172]) Added a `types` query parameter for listing [authentication providers][176] and [secrets providers][177] via the API.
+- ([Commercial feature][172]) The Alpine-based Docker image now has multi-architecture support with support for the linux/386, linux/amd64, linux/arm64, linux/arm/v6, linux/arm/v7, linux/ppc64le, and linux/s390x platforms.
+- The backend flag [`--api-request-limit`][173] is now available to configure the maximum API request body size (in bytes).
+- In the [REST API][174], most configuration resources now support the PATCH method for making updates.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][172]) Improved logging for OIDC authentication providers. Also added [`disable_offline_access` OIDC spec attribute][180], which allows OIDC providers to return a refresh token so users can refresh their access tokens.
```

Comment:
I would maybe change the description for `disable_offline_access`, to mention that it's basically a workaround for authorization servers that do not support the `offline_access` scope.